TempleEight/docs/guide/open-api.mdx

```diff
@@ -0,0 +1,208 @@
+
+Then select Single View App:
+<img alt="Create a new iOS App" src={useBaseUrl('img/xcode-new-ios.png')} />
+
+Fill in your project details as necessary and save the project:
+<img alt="Create a new iOS App" src={useBaseUrl('img/xcode-new-ios-detail.png')} />
+
+Now - close the Xcode window that just appeared and reopen your terminal.
+Move to your Xcode project directory and enter the following command:
+
+```bash
+# Move to your Xcode project directory
+❯❯❯ cd ~/Desktop/ExampleClient
+
+# Create a new Cocoapod project
+❯❯❯ pod init
+```
+
+This will have created a `Podfile` similar to the following:
+
+
+```
+# Uncomment the next line to define a global platform for your project
+# platform :ios, '9.0'
+
+target 'ExampleClient' do
+  # Comment the next line if you don't want to use dynamic frameworks
+  Teuse_frameworks!
+
+  # Pods for ExampleClient
+
+end
+```
+
+We're going to add 2 dependencies to our Podfile: one for [Alamofire](https://github.com/Alamofire/Alamofire), a HTTP library, and another to our generated client library.
+The path to the generated client library must match the `-o` argument to `openapi-generator` from the previous step.
+
+
+```
+# Uncomment the next line to define a global platform for your project
+# platform :ios, '9.0'
+
+target 'ExampleClient' do
+  # Comment the next line if you don't want to use dynamic frameworks
+  use_frameworks!
+
+  # Pods for ExampleClient
+  pod 'Alamofire', '~> 4.9.0'
+  pod 'OpenAPIClient', :path => '~/Desktop/example-openapi'
+
+end
+```
+
+Following this, install the dependencies with:
+
+```bash
+# Install the Pod dependencies 
+❯❯❯ pod install
+```
+
+This will instruct you to use the `.xcworkspace` generated in this directory from now on, which we will now open in Xcode:
+
+<img alt="Xcode workspace" src={useBaseUrl('img/xcode-workspace.png')} />
+
+All of the code will be written in this class from now on, but if you're a Swift expert, feel free to create your own.
+
+## Starting the server for local development
+
+Before we get started writing our client, we need to deploy the Temple project for local development.
+This can be done by following the [Getting Started](../getting-started) guide, or from the root of the generated Temple project folder, by running the following commands:
+
+
+```bash
+# Deploy the services for local development
+❯❯❯ source deploy.sh
+
+# Find the base URL to make requests to
+❯❯❯ echo $KONG_ENTRY
+localhost:8000
+```
+
+This second step gave us the base URL where requests to our API can be made.
+
+## Using the Client API
+
+Now, back in the Xcode project, we can tell the client where to make requests by settings the `basePath` property of the client, like so:
+
+
+```swift
+import UIKit
+import OpenAPIClient
+
+class ViewController: UIViewController {
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        OpenAPIClientAPI.basePath = "http://localhost:8000/api"
+    }
+}
+```
```

Comment:
If there is someway to syntax highlight/bold out the lines to change, would be good